code/modules/cargo/randomstock.dm

```diff
@@ -74,8 +74,6 @@ var/list/global/random_stock_common = list(
 	"light" = 1.8,
 	"aid" = 4,
 	"flame" = 2,
-	"crayons" = 1.5,
-	"figure" = 1,
```

Comment:
Against removing those things from cargo, not everything must have an use, as plushies and toys have some rp value, mostly as gifts.